ui/app/templates/components/job-page.hbs

```diff
@@ -19,7 +19,7 @@
         "job-page/parts/latest-deployment" job=@job handleError=this.handleError
       )
       TaskGroups=(component "job-page/parts/task-groups" job=@job)
-      RecentAllocations=(component "job-page/parts/recent-allocations" job=@job)
+      RecentAllocations=(component "job-page/parts/recent-allocations" job=@job activeTask=@activeTask setActiveTaskQueryParam=@setActiveTaskQueryParam)
```

Comment:
This, and further job-page/*.hbs file changes, were tricky to figure out how to best handle (and thank you to @DingoEatingFuzz for unblocking me!)

The nature of our higher-order components is such that, to get query parameters at the /job level, we have to declare our passed variables more verbosely than desired.